### PR TITLE
Bumped GATK4 version

### PR DESCRIPTION
**Pull request by micknudsen**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/gatk4/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ build:
 source:
   fn: gatk-{{ version }}.zip
   url: https://github.com/broadinstitute/gatk/releases/download/{{ version }}/gatk-{{ version }}.zip
-  md5: 87b173e3abde88de2d9fcb601b4902e8
+  md5: 1b4c7ade9aa3740a3adde6922ec35ca0
```
Comment (mbargull) [inline recipes/gatk4/meta.yaml]: We should update this to `sha256` at some point. Do you want to do this now?

Comment (mbargull) [reply recipes/gatk4/meta.yaml]: That'd be
```yaml
  sha256: 11abae169b73946d52e7831d8ada8b01ab05c517bb60abbf4046eeed3c696065
```

Comment (micknudsen) [reply recipes/gatk4/meta.yaml]: Didn't see that before it was too late. Next time I'm making a pull request, I will make sure to replace MD5s by SHA256s.

**Comment by micknudsen** [comment]
Ah, it wasn't too late to add the SHA256. I thought that the pull request was already merged.

**Comment by mbargull** [comment]
No worries -- I took that as an opportunity to test out GitHub's web-based editor 😉. Will merge as soon as the new commit is built.

**Comment by mbargull** [comment]
Thanks for the update @micknudsen!

**Pull request merged**